app/views/builds/_builds_table.html.erb

```diff
@@ -0,0 +1,27 @@
+<section class="tabs">
+  <p class="pull-right">
+    <%= link_to "Create Build", new_project_build_path(@project), class: "btn btn-default" %>
+  </p>
+
+  <table class="table">
+    <thead>
+    <tr>
+      <% unless @project %>
+          <th>Project</th>
+      <% end %>
+      <th>Id</th>
```

Comment:
probably don't need this. It's a samson internal value and it's available in the URL to anyone that really needs it